app/models/measurement.rb

```diff
@@ -10,11 +9,8 @@ class Measurement < ActiveRecord::Base
 
   #input: measurement
   def bmi_calc
-    #first = last, because of ordering in model
-    weight = self.weight
-    height = self.person.length
+    height = person.length
```

Comment:
Not sure if I understand this. When calculating process data, is it far more expensive with the  person.height stored in a var and then calculated, than it is with person.height called directly in the calculation?